Classes/Typo3PageIndexer.php

```diff
@@ -315,7 +315,10 @@ protected function getPageDocument()
         $document->setField('rootline', $rootline);
 
         // access
-        $document->setField('access', (string)$this->pageAccessRootline);
+        $access = (string)$this->pageAccessRootline;
+        if (trim($access) !== "") {
```

Comment:
This should never be empty. The access field always needs a value of at least `c:0` or `r:0`